rules/prefer-event-target.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const MESSAGE_ID = 'prefer-event-target';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `EventTarget` over `EventEmitter`.',
+};
+
+const eventEmitterClassSelector = [
+	':matches(ClassDeclaration, ClassExpression)',
+	'[superClass.name="EventEmitter"]',
+	'[body.type="ClassBody"]',
+].join('');
```

Comment:
```suggestion
const eventEmitterSuperClassSelector = [
	':matches(ClassDeclaration, ClassExpression)',
	'[body.type="ClassBody"]',
	' > ',
	'[superClass.name="EventEmitter"]',
].join('');
```

So we can report on the super class instead of the whole class.